src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/ImplicitlyTriggeredDebugBuildManager.cs

```diff
@@ -13,14 +12,13 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Build
 {
     /// <summary>
-    /// Tracks implicitly builds triggered as part of F5/Shift+F5 debugging commands and
-    /// skips analyzer execution for these builds by invoking into <see cref="IImplicitlyTriggeredBuildManager"/>.
+    /// Tracks implicitly builds triggered as part of F5/Ctrl+F5 debug/launch commands and
+    /// updates the <see cref="IImplicitlyTriggeredBuildManager"/> appropriately.
     /// </summary>
     [Export(ExportContractNames.Scopes.UnconfiguredProject, typeof(IProjectDynamicLoadComponent))]
     [AppliesTo(ProjectCapability.DotNet)]
     internal class ImplicitlyTriggeredDebugBuildManager : OnceInitializedOnceDisposedAsync, IProjectDynamicLoadComponent, IVsUpdateSolutionEvents2, IVsUpdateSolutionEvents3
```

Comment:
Do we want to rename this type to remove “Debug” in the name, given you plan to use this type for other kinds of implicit builds in the near future?

Reply:
The type will continue to be specific to F5/Ctrl+F5 scenarios, though I'm going to update it to retrieve other information related to those scenarios such as the set of startup projects.

I did think about renaming it as we often use "Debug" when we're really referring to launching a project, with or without the debugger. If I do that I'll want to update a bunch of type names.